apps/sms/js/activity_handler.js

```diff
@@ -241,7 +246,11 @@ var ActivityHandler = {
         return;
       }
 
-      Utils.confirm('discard-new-message').then(() => {
+      Utils.confirm(
+        'discard-new-message',
+        'unsent-message-title',
```

Comment:
not sure what title will be better here (default or the same for all similar cases), what do you think?